android/build.gradle

```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.3"
-        minSdkVersion = 18
+        minSdkVersion = 19
```

Comment:
Why is this needed? Some people have quite old devices, see #745

Reply:
Ok sorry, I didn't see the comment of 2373df7
I see, it appeared in [version 4.29.0](https://github.com/mercadopago/px-android/releases/tag/4.29.0)

Reply:
Another possibility would be to override the version, like:

```
mercadoPagoPxVersion = "4.28.0"
```

But it doesn't seem like a good idea... @jamesdgrant are there lots of messengers with Android 4.3?